### ISSUE TITLE
GO:0070197 needs text edits

### ISSUE THREAD

**Issue by mah11** [geneontology/go-ontology#12072]
GO:0070197 attachment of telomere to nuclear envelope

The name and text def need to make it explicit that this term is specific for the meiotic cell cycle, as its asserted and inferred parents imply. Maybe "attachment of telomere to nuclear envelope" for the nome? Also review the synonyms.

It's probably also worth considering how GO:0070197 and GO:0044821 'meiotic telomere tethering at nuclear periphery' should connect -- I don't know enough to say whether they should be merged, or one should be part_of the other, or what.

[Term]
id: GO:0070197
name: attachment of telomere to nuclear envelope
namespace: biological_process
def: "The cell cycle process in which physical connections are formed between telomeric heterochromatin and the nuclear envelope, facilitating bouquet formation." [GOC:jp, GOC:pr, GOC:vw, PMID:18818742]
synonym: "attachment of telomeres to nuclear envelope" BROAD []
synonym: "attachment of telomeric chromatin to nuclear envelope" BROAD []
synonym: "attachment of telomeric heterochromatin to nuclear envelope" EXACT []
is_a: GO:0097240 ! chromosome attachment to the nuclear envelope
is_a: GO:1903046 {is_inferred="true"} ! meiotic cell cycle process
relationship: part_of GO:0045141 ! meiotic telomere clustering

I hope the "quick fix" label doesn't turn out to be wishful thinking! ;)

**Comment by mah11** [comment]
@ValWood

**Comment by ukemi** [comment]
I made it a part_of GO:0044821 for now. If we decide they are synonymous, then we can merge. I did the same name and def change with the spindle pole body term based on its position in the graph.

[Term]
id: GO:0070197
name: meiotic attachment of telomere to nuclear envelope
namespace: biological_process
def: "The meiotic cell cycle process in which physical connections are formed between telomeric heterochromatin and the nuclear envelope, facilitating bouquet formation." [GOC:jp, GOC:pr, GOC:vw, PMID:18818742]
synonym: "attachment of telomeres to nuclear envelope" BROAD []
synonym: "attachment of telomeric chromatin to nuclear envelope" BROAD []
synonym: "attachment of telomeric heterochromatin to nuclear envelope" EXACT []
is_a: GO:0097240 ! chromosome attachment to the nuclear envelope
is_a: GO:1903046 {is_inferred="true"} ! meiotic cell cycle process
relationship: part_of GO:0044821 ! meiotic telomere tethering at nuclear periphery
